### DISCUSSION TITLE
GitHub Like a Boss 🍩 MishManners 🍩 Ask Your Questions here!

### DISCUSSION THREAD

**Discussion by mishmanners** [github/india#97] [Open Source]
What did you think of my talk?
Any questions you have for me?

**Comment by EJL3** [comment]
The session was cool!! @MishManners

**Reply by mishmanners** [reply]
Thank you so much! Glad you liked it 😄

**Reply by NDBEST23** [reply]
I don't understand how this GitHub works and how it can help my life

**Reply by mishmanners** [reply]
Depends on what you want to do with it. What do you do for work @NDBEST23?

**Reply by NDBEST23** [reply]
I'm an engineer. Mechanical engineer

**Reply by EJL3** [reply]
@NDBEST23 its simple, Github is a open source where u can find projects of the language you prefer. for more visit: https://guides.github.com/activities/hello-world/

**Reply by NDBEST23** [reply]
Sorry for the disturb. I followed the step instructed by the link u sent .but I did not  see the NEW REPOSITORY option

**Reply by EJL3** [reply]
@NDBEST23 u can find that in ur profile on the top right section showing a + sign

**Reply by NDBEST23** [reply]
![Screenshot_20210327-094031.png](https://user-images.githubusercontent.com/80988100/112715461-91828280-8ee0-11eb-95b0-e24d96721b37.png)

**Reply by NDBEST23** [reply]
![Screenshot_20210327-094205.png](https://user-images.githubusercontent.com/80988100/112715493-be369a00-8ee0-11eb-83a9-f6f7f5f2b905.png)

**Reply by EJL3** [reply]
@NDBEST23 it would be good if u do this in a pc

**Reply by NDBEST23** [reply]
If I have one. I can't afford a computer

**Reply by mishmanners** [reply]
The introduction to GitHub one is the best to get started with: https://lab.github.com/

**Reply by NDBEST23** [reply]
Thanks once again

**Comment by HUSK1E** [comment]
Great session. What's another top tip you have for us? You mentioned we could be here for hours talking about top tips. Hit us with another one.

**Reply by mishmanners** [reply]
Oooo I got one for you! Keyboard shortcuts. My favourite is using "T" to find a file in a repo. There's heaps of them. I even did a short video on Twitter about keyboard shortcuts:
https://twitter.com/github/status/1369347441724645377

**Comment by EJL3** [comment]
Also will there be a Arctic code vault archive pgm soon for the users who missed the arctic badge?

**Reply by mishmanners** [reply]
The Artic Code Vault is such an amazing program. You can read more about it here: https://archiveprogram.github.com/
The code was locked on 02/02/2020.

**Reply by EJL3** [reply]
Thank you

**Reply by mishmanners** [reply]
Pleasure. Linking to this Discussion so we know it's been answered: https://github.com/github/india/discussions/101

**Reply by EJL3** [reply]
Guess i have to thank u again😀

**Comment by tsadarsh** [comment]
Can you please view #99.

**Reply by mishmanners** [reply]
Yep answered 😄
